django_react_proj/users-fe/src/components/SummingTime/SummingTime.js

```diff
@@ -17,7 +17,7 @@ const Subtract = ({ users }) => {
   }
   const taskhours = Math.floor(timevalue / 60);
   const taskminutes = timevalue % 60;
```

Comment:
Why do yo uneed to cast boolean to string? You used it in comparions to `'true'` which makes no sense. Just use boolean

Reply:
I didn't notice that. Changed.